### PR TITLE
design-docs: timetable v2

### PR DESCRIPTION
**Pull request by multun**

### CODE REVIEW

File: content/docs/reference/design-docs/timetable.en.md
Comment (Khoyo) [inline content/docs/reference/design-docs/timetable.en.md:16]:
```suggestion
## Modifiable fields
```

**Review by flomonster — APPROVED** [review]

File: content/docs/reference/design-docs/timetable.en.md
Comment (clarani) [inline content/docs/reference/design-docs/timetable.en.md:82]:
```suggestion
# takes a path (the output of pathfinding/blocks),
```

**Comment by multun** [comment]
@flomonster @clarani @Khoyo thanks for the review :) I've submitted an amendment to this proposal, which changes how margins and arrival times are defined https://github.com/osrd-project/osrd-website/pull/164 before merging

**Comment by flomonster** [comment]
## Note meeting with PO (Alex + Giuliana)

- Are engineering margins useful (can be defined by scheduled points)?
  - If not, we can get rid of it.
- We need the standard margin system whatsoever. It compromises the suggestion here: https://github.com/osrd-project/osrd-website/pull/164

File: content/docs/reference/design-docs/timetable.en.md
Comment (flomonster) [inline content/docs/reference/design-docs/timetable.en.md:52]: Wait for @bgiuliana confirmation.

Comment (flomonster) [reply content/docs/reference/design-docs/timetable.en.md:52]: @bgiuliana Verified this information.

File: content/docs/reference/design-docs/timetable/index.en.md
Comment (flomonster) [inline content/docs/reference/design-docs/timetable/index.en.md:99]: Ask @bgiuliana about it.

**Review by flomonster — APPROVED** [review]
Great!

**Review by flomonster — APPROVED** [review]

**Pull request merged**
